### PR TITLE
Upgrade k8s resource versions 

### PR DESCRIPTION
**Pull request by Pooja-08**

### CODE REVIEW

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-2 failed.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-1 is succeeded.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-2 failed.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n+n+n is succeeded.

File: helm-charts/interoperator/templates/broker.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ spec:
   selector:
     app: {{ .Release.Name }}-broker
 ---
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1
```
Comment (saud89) [inline helm-charts/interoperator/templates/broker.yaml:21]: Hi @Pooja-08, please check if this apiversion is supported in k8s versions 1.18 and 1.19 also...

**Comment by anoopjb** [comment]
Superseded by #1472

**Pull request closed without merging**